project-koku/koku
# COST-430: Add support for tag based rates in cost model API.

Jira Issue: 
- https://issues.redhat.com/browse/COST-430

This pr is to add support for the tag based rates.

#### How to test

I have already added an example tag rate to the `openshift_on_prem_cost_model.json`. 

I think it would be beneficial to add a couple more scenarios. Like multiple of the same tag key, multiple tag values, a rate for infrastructure and a rate for supplementary. 

After you have finished modifying the json you can then run:
`make load_test_customer_data` to populate the database and upload the cost model.  

Then you will want to check to see if the tag rates show up: 
- http://localhost:8000/api/cost-management/v1/cost-models/
